.github/ISSUE_TEMPLATE.md

```diff
@@ -1,37 +1,27 @@
 <!--
-Questions:
-  https://groups.google.com/forum/#!forum/loopbackjs
-  https://gitter.im/strongloop/loopback
-Immediate support:
-  https://strongloop.com/api-connect-faqs/
-  https://strongloop.com/node-js/subscription-plans/
+HELP US HELP YOU BY PROVIDING AS MUCH OF THE FOLLOWING INFORMATION AS POSSIBLE AND
```

Comment:
I understand that we are trying to emphasize what's being said here, but the all caps seems like we're screaming at them :P. Can we make this statement bold instead?

Reply:
Can't do bold since this is a comment. But I do see your point and have modified it ... to some extent.